src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,18 @@
+.stars {
+  display: flex;
+
+  &__star {
+    height: 16px;
+    width: 16px;
+    margin-right: 4px;
+    background-image: url(../images/star.svg);
+    background-repeat: no-repeat;
+    background-position: center;
+  }
+}
+
+@for $starCount from 1 through 5 {
+  .stars--#{$starCount} :nth-child(-n + #{$starCount}) {
+    background-image: url(../images/star-active.svg);
```

Comment:
Same as above, change the double quotes to single quotes for consistency: url('../images/star-active.svg')